vmlifecycle/vmlifecyclecommands/upgrade_opsman_test.go

```diff
@@ -454,43 +457,43 @@ var _ = Describe("upgradeOpsman", func() {
-							table.Entry("semver with build vs semver, via patch", "2.5.2", "2.5.3-build.0"),
-							table.Entry("semver with build vs semver, via minor", "2.4.3", "2.5.3-build.0"),
-							table.Entry("semver with build vs semver, via major", "1.5.3", "2.5.3-build.0"),
+							Entry("semver vs semver with build, via patch", "2.5.2-build.0", "2.5.3"),
+							Entry("semver vs semver with build, via minor", "2.4.3-build.0", "2.5.3"),
+							Entry("semver vs semver with build, via major", "1.5.3-build.0", "2.5.3"),
+							Entry("semver with build vs semver, via patch", "2.5.2", "2.5.3-build.0"),
+							Entry("semver with build vs semver, via minor", "2.4.3", "2.5.3-build.0"),
+							Entry("semver with build vs semver, via major", "1.5.3", "2.5.3-build.0"),
 
 							// Semver vs. build
-							table.Entry("semver vs build, via patch", "2.5-build.0", "2.5.1"),
-							table.Entry("semver vs build, via minor", "2.4-build.1", "2.5.1"),
-							table.Entry("semver vs build, via major", "1.5-build.1", "2.5.1"),
-							table.Entry("build vs semver, via patch", "2.5.0", "2.5-build.1"),
-							table.Entry("build vs semver, via minor", "2.4.1", "2.5-build.1"),
-							table.Entry("build vs semver, via major", "1.5.1", "2.5-build.1"),
+							Entry("semver vs build, via patch", "2.5-build.0", "2.5.1"),
+							Entry("semver vs build, via minor", "2.4-build.1", "2.5.1"),
+							Entry("semver vs build, via major", "1.5-build.1", "2.5.1"),
+							Entry("build vs semver, via patch", "2.5.0", "2.5-build.1"),
+							Entry("build vs semver, via minor", "2.4.1", "2.5-build.1"),
+							Entry("build vs semver, via major", "1.5.1", "2.5-build.1"),
 
 							// Semver with build vs build
-							table.Entry("semver with build vs build, via patch", "2.5-build.0", "2.5.1-build.103"),
-							table.Entry("semver with build vs build, via minor", "2.4-build.1", "2.5.1-build.103"),
-							table.Entry("semver with build vs build, via major", "1.5-build.1", "2.5.1-build.103"),
-							table.Entry("build vs semver with build, via patch", "2.5.0-build.103", "2.5-build.1"),
-							table.Entry("build vs semver with build, via minor", "2.4.1-build.103", "2.5-build.1"),
-							table.Entry("build vs semver with build, via major", "1.5.1-build.103", "2.5-build.1"),
+							Entry("semver with build vs build, via patch", "2.5-build.0", "2.5.1-build.103"),
+							Entry("semver with build vs build, via minor", "2.4-build.1", "2.5.1-build.103"),
+							Entry("semver with build vs build, via major", "1.5-build.1", "2.5.1-build.103"),
+							Entry("build vs semver with build, via patch", "2.5.0-build.103", "2.5-build.1"),
+							Entry("build vs semver with build, via minor", "2.4.1-build.103", "2.5-build.1"),
+							Entry("build vs semver with build, via major", "1.5.1-build.103", "2.5-build.1"),
 						)
 					}
 
-					It("polls till the provided timeout then errors", func(done Done) {
+					It("polls till the provided timeout then errors", func(ctx context.Context) {
```

Comment:
Is the Context in the test function necessary? It doesn't seem to be used.

Reply:
explained in the [previous](https://github.com/pivotal-cf/om/pull/661#discussion_r1698970284) comment